src/patternfly/components/Nav/examples/Navigation.md

```diff
@@ -8,6 +8,91 @@ import './Navigation.css'
 
 ## Examples
 
+### New spacing
+```hbs
+{{#> nav nav--id='new-spacing-example' nav--aria-label='Global'}}
+  {{#> nav-section nav-section-title--text='Cluster'}}
+    {{#> nav-list}}
+      {{> nav-item nav-link nav-link--text='Dashboard'}}
+      {{> nav-item nav-link nav-link--text='Builds'}}
+      {{> nav-item nav-link nav-link--text='Compute'}}
+      {{> nav-item nav-link nav-link--text='Networking'}}
+      {{> nav-item nav-link nav-link--text='Observe'}}
+      {{> nav-item nav-link nav-link--text='Operators'}}
+      {{> nav-item nav-link--IsCurrent=true nav-link--text='Applications'}}
+      {{> nav-item nav-link nav-link--text='Workloads'}}
+    {{/nav-list}}
+  {{/nav-section}}
+  {{#> nav-section nav-section-title--text='Settings'}}
+    {{#> nav-list}}
+      {{> nav-item nav-link nav-link--text='User management'}}
+      {{> nav-item nav-link nav-link--text='Administration'}}
+      {{> nav-item nav-link nav-link--text='Import YAML'}}
+      {{> nav-item nav-link nav-link--text='Help & support'}}
+    {{/nav-list}}
+  {{/nav-section}}
+{{/nav}}
+```
+
+## New Expandable
```

Comment:
```suggestion
### New Expandable
```
Hierarchy in the headings here - but also, what should these really be called? (probably not "New" right?)

Reply:
Ah, yes. Those examples are for demo purposes. They'll be removed before merge.